IBM Cloud/WML/notebooks/watsonx/LLM Metrics Evals-VertexAI-OpenPages.ipynb

```diff
@@ -34,7 +34,7 @@
    "outputs": [],
    "source": [
     "!pip install -i https://test.pypi.org/simple/ ibm-watson-openscale==3.0.34.8  | tail -n 1\n",
-    "!pip install -i https://test.pypi.org/simple/ ibm-metrics-plugin==4.8.1.13  | tail -n 1"
+    "!pip install -i https://test.pypi.org/simple/ ibm-metrics-plugin==5.0.1.17  | tail -n 1"
```

Comment:
Why do we have `5.0.1.17` here instead of `5.0.1.16` used in above notebooks?

Reply:
Referring from [here](https://github.ibm.com/aiopenscale/tracker/issues/36372#issuecomment-84524021), We are using `5.0.1.17` version for `ibm-metrics-plugin` and `5.0.1.16` for `ibm_wos_utils`

Reply:
sorry, I got confused. thanks for the info